### DISCUSSION TITLE
How do I see a list of persons that are Trusted in a plot?

### DISCUSSION THREAD

**Discussion by ElMoha943** [TownyAdvanced/Towny#5169] [Q&A] [answered]
how do i see a list of persons that are trusted in a plot?

**Comment by Warriorrrr** [comment, marked as answer]
Trusted players are displayed in the /plot perm screen

**Comment by Seagoddess** [comment]
/plot perm screen is NOT a command

The only /plot perm commands are:
/plot perm hud
/plot perm remove 
/plot perm add
/plot perm gui

**Reply by LlmDl** [reply]
Use only `/plot perm`, no screen on the end. The result you will be shown is what we call the "plot perm screen".
